src/sentry/issues/merge.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from typing import Dict, Sequence
+from uuid import uuid4
+
+import rest_framework
+
+from sentry import eventstream
+from sentry.models import Activity, Group, GroupCategory, GroupStatus, Project, User
+from sentry.tasks.merge import merge_groups
+from sentry.types.activity import ActivityType
+
+
+def handle_merge(
+    group_list: Sequence[Group],
+    project_lookup: Dict[int, Project],
+    acting_user: User | None,
+) -> Dict[str, str | list[str]]:
```

Comment:
Let's return a `TypedDict` here, since we know what the keys in this dict will be